.circleci/config.yml

```diff
@@ -464,42 +650,55 @@ workflows:
             branches:
               only: master
 
+  # release-branch workflow is triggered when branch is "release-*"
+  # and tag is not empty
   release-branch:
+    when:
+      and:
+        - matches:
+            pattern: *release-branch-regex
+            value: << pipeline.git.branch >>
+        - matches:
+            pattern: ^\S+$
```

Comment:
adding `when` to the workflow intended to replace the filters on each job, so this pattern was intended to be the "match" to exclude any tag, but now i see it might not be correct. 

```
filters:
            branches:
              only: *release-branch-regex
            tags:
              ignore: /.*/ <-- this one, to ignore any tag? 
```